app/views/podcast_planner/_calendar.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="col" data-controller="calendar">
+  <%= month_calendar(start_date: month.first) do |date| %>
+    <% if date_is_within_month(date, month) %>
+      <%= date.day %>
+
+      <% if is_preselected_date?(date, month) %>
```

Comment:
feels a little wordy, but essentially pre-selects the hidden_field if it's one of the calculated dates, and disables the field if not.

Reply:
+1 makes sense to me.

You could also drop the if/else, and just make this `form.hidden_field ..., disable: !is_preselected_date?`